bears/general/coalaBear.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.results.Diff import Diff
```

Comment:
not really sure whether this is a worthy change, but according to alphabetical imports, this line should go up.